services/checks/checks.go

```diff
@@ -1052,26 +1054,20 @@ func (s *Service) loadLocalChecks(file string) ([]check.Check, error) {
 func (s *Service) downloadChecks(ctx context.Context) ([]check.Check, error) {
 	s.l.Infof("Downloading checks from %s ...", s.host)
 
-	settings, err := models.GetSettings(s.db)
+	ssoDetails, err := models.GetPerconaSSODetails(ctx, s.db.Querier)
```

Comment:
what if SSO params are not set (PMM Server is not connected to Portal) ?

Checks should continue work with(connected to Portal) and without(anonymous) SSO credentials